src/ansys/conceptev/core/progress.py

```diff
@@ -100,4 +113,4 @@ def monitor_job_progress(job_id: str, user_id: str, token: str):
     msal_app = create_msal_app()
     token = get_ansyId_token(msal_app)
     user_id = get_user_id(token)
-    monitor_job_progress(job_id, user_id, token)
+    monitor_job_progress(job_id, user_id, token, timeout=1)
```

Comment:
Would it make sense to extend this timeout a bit ?

Reply:
pushed some new change that make this fully configurable. Also added an extra function I needed for grabbing some extra data. Sorry for the scope creep on this PR. I realise best practice would be to create a new PR but not this time ;)